libs/island-ui/core/src/lib/RadioButton/RadioButton.tsx

```diff
@@ -114,7 +114,7 @@ export const RadioButton = ({
           <Text as="span" fontWeight={checked ? 'semiBold' : 'light'}>
             {label}
           </Text>
-          {subLabel && large && (
```

Comment:
Is sublabel only visible if the component is large as well?

Reply:
I removed this so that we could have a sublabel on smaller radiobuttons as well, so it's just visible if there is any content in the sublabel parameter